### ISSUE TITLE
core-2021.8.0b0 - cannot import name 'VOLT'

### ISSUE THREAD

**Issue by IonutNeagu** [custom-components/ble_monitor#452]
`Unable to prepare setup for platform ble_monitor.sensor: Platform not found (cannot import name 'VOLT' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)).`

**Comment by Ernst79** [comment]
Thanks, will look into it. Seems to be a change in the HA beta.

**Comment by airy10** [comment]
See https://github.com/home-assistant/core/pull/53243

**Comment by Ernst79** [comment]
Fixed in 4.3.1, including backwards compatibility.

**Comment by robtheone** [comment]
Is this backwards compatibility still working, because last night after installing ble_monitor version 4.6.2 I was getting the same error message:
 (Unable to prepare setup for platform ble_monitor.sensor: Platform not found (cannot import name 'VOLT' from homeassistant.const')
When I looked the error up, I see that in version 4.3.1 the issue was resolved and shows that it included backwards compatibility.  I had to uninstall 4.6.2 completely, then installed 4.3.1 and everything has been working smoothly.
I am still running core-2021.6.6 because the update to core-2021-8.x broke other energy monitoring dashboards, so I had to revert back.  Thank you
